manifests/job.pp

```diff
@@ -213,8 +213,12 @@
     fail("Backup resource ${title} cannot contain whitespace or special characters")
   }
 
-  $storage_tags = [ $_env_tag, $storage ]
-  $readonly_tags = [ $_env_tag, $worker_tag, 'zpr_vol' ]
+  if $storage == undef {
```

Comment:
@danielparks, were you encountering a case where this was showing up as undef? The parameter is required so I'm curious to understand better.